### DISCUSSION TITLE
Custom filters

### DISCUSSION THREAD

**Discussion by CabalCrow** [gokcehan/lf#1701] [Ideas]
I was reading through this closed pull request [#763](https://github.com/gokcehan/lf/pull/763) about adding | to filters. From what I understand the main issue was that it ends up making fliters too complex & require too many options.
What I was wondering was if this could actually be something that can be handled from outside of lf, by giving an option to select a custom program to filter the results? Something like lf outputing the current directory files & the filter string to another program specified by an option & expecting a filtered down input. This could also potentially double as a way to do custom sorting as I've seen this requested as well.

**Comment by joelim-work** [comment]
I think custom filtering can be done at the same time as custom sorting. Someone else was working on it in #618, perhaps you can request filtering there.
